metrics/test_driver/common/process_manager/process_manager.dart

```diff
@@ -7,14 +7,16 @@ import '../../util/file_utils.dart';
 import '../command/command.dart';
 import '../config/browser_name.dart';
 import '../config/device.dart';
+import 'process/flutter_process.dart';
+import 'process/process_wrapper.dart';
+import 'process/selenium_process.dart';
 
-/// The class needed to manage started processes and start new processes
+/// Starts new processes and then manages them.
```

Comment:
Let's review this one more time together, I want to make sure that we are doing everything right in this class.